.Net Editor does not support System.Range

Reproduce:
* click Run button on first code block
* copy code from last code block to editor
* click Run on editor  

Error:
(6,5): error CS0246: The type or namespace name 'Range' could not be found (are you missing a using directive or an assembly reference?)
(6,15): error CS0518: Predefined type 'System.Index' is not defined or imported
(6,15): error CS8652: The feature 'range operator' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.
(6,15): error CS0518: Predefined type 'System.Range' is not defined or imported
(6,22): error CS0518: Predefined type 'System.Index' is not defined or imported
(13,5): error CS0246: The type or namespace name 'Range' could not be found (are you missing a using directive or an assembly reference?)
(13,15): error CS8652: The feature 'index operator' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.
(13,15): error CS0518: Predefined type 'System.Index' is not defined or imported
(13,15): error CS0656: Missing compiler required member 'System.Index..ctor'
(13,15): error CS0518: Predefined type 'System.Index' is not defined or imported
(13,15): error CS8652: The feature 'range operator' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.
(13,15): error CS0518: Predefined type 'System.Range' is not defined or imported
(13,30): error CS8652: The feature 'index operator' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.
(13,30): error CS0518: Predefined type 'System.Index' is not defined or imported
(13,30): error CS0656: Missing compiler required member 'System.Index..ctor'
(13,30): error CS0518: Predefined type 'System.Index' is not defined or imported
(18,69): error CS0246: The type or namespace name 'Range' could not be found (are you missing a using directive or an assembly reference?)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a92f2fd3-888f-340e-72dd-8a3338313783
* Version Independent ID: 67981ff3-5016-3147-b3c2-5bc16efe1416
* Content: [Explore ranges of data using indices and ranges](https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/ranges-indexes#feedback)
* Content Source: [docs/csharp/tutorials/ranges-indexes.md](https://github.com/dotnet/docs/blob/master/docs/csharp/tutorials/ranges-indexes.md)
* Product: **dotnet-csharp**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

--- a/docs/csharp/tutorials/ranges-indexes.md
+++ b/docs/csharp/tutorials/ranges-indexes.md
@@ -27,7 +27,7 @@ This language support relies on two new types and two new operators:
 
 Let's start with the rules for indices. Consider an array `sequence`. The `0` index is the same as `sequence[0]`. The `^0` index is the same as `sequence[sequence.Length]`. Note that `sequence[^0]` does throw an exception, just as `sequence[sequence.Length]` does. For any number `n`, the index `^n` is the same as `sequence[sequence.Length - n]`.
 
-```csharp-interactive
+```csharp
 string[] words = new string[]
 {
                 // index from start    index from end

